src/output/plugins/OSXOutputPlugin.cxx

```diff
@@ -795,6 +795,7 @@ OSXOutput::Open(AudioFormat &audio_format)
 	PcmExport::Params params;
 	params.alsa_channel_order = true;
```

Comment:
btw. unrelated to this commit, not your code.... why is ALSA channel order only used with DoP? Does OSX always need ALSA channel order or never? It doesn't appear to make sense to have it only with DoP.

Reply:
No idea what alsa's channel order is. We allow users to configure their channel map via https://github.com/MusicPlayerDaemon/MPD/blob/1ffc2c6c5d0c3f6a09059de21762bb203d46d809/src/output/plugins/OSXOutputPlugin.cxx#L291